lib/stats/stat.js

```diff
@@ -0,0 +1,106 @@
+import assert from 'assert';
+import base from 'taskcluster-base';
+
+import * as series from './series';
+
+export default class Stat {
+  constructor(config) {
+    assert(config.influx, 'Must supply an influx configuration');
+    assert(config.workerId, 'Worker ID is required');
+    assert(config.workerType, 'Worker type is required');
+    assert(config.workerGroup, 'Worker group is required');
+    assert(config.workerNodeType, 'Worker instance type is required');
+    assert(config.provisionerId, 'Provisioner ID is required');
+
+    this.influx = new base.stats.Influx(config.influx);
+    this.baseWorkerStats = {
+      workerId: config.workerId,
+      workerGroup: config.workerGroup,
+      workerType: config.workerType,
+      instanceType: config.workerNodeType,
+      provisionerId: config.provisionerId,
+      capacity: config.capacity
+
+    };
```

Comment:
I  see how this makes sense... How about we modify `base.stats.Series#reporter`, such that we can do the following:
```js
var mySeries = new base.stats.Series({.....});
var drain = new base.stats.Influx({....});

var reportMySeries = mySeries.reporter({
  drain: drain,
  tags: {
    workerId: '...',
    workerGroup: '...',
    ....
  }
});
```

Part of the idea with `Series` was that things could be more typed, instead of using strings as identifiers, which is what the `record` method does here. Hence instead of `stats.record('mySeries', value)` we do `stats.reportMySeries({value: ....})`.

Adding this to `base.Series` should be easy... ping me if you like the idea... I think I can already see one, possibly two, uses cases for this in `base.stats` already...